HealthcareSystem/EventSourcingService/Controllers/EventController.cs

```diff
@@ -9,11 +9,11 @@ namespace EventSourcingService.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
-    public class EventStoreExampleController : ControllerBase
+    public class EventController : ControllerBase
```

Comment:
I would like a more specific name for this controller as well, there will be many events and they can't all have the 'api/event' endpoint.